server/models/Profile.js

```diff
@@ -0,0 +1,40 @@
+import mongoose from 'mongoose'
+
+const profileSchema = mongoose.Schema({
+	profile_photo: {
+    type: String,
+  },
+	first_name: {
+    type: String,
+    required: true,
+  },
+	last_name: {
+    type: String,
+    required: true,
+  },
+	gender: {
+    type: String,
+    enum: ['male', 'female'],
+  },
+	birth_date: {
+    type: Date,
+    required: true,
+  },
+	phone_number: {
+    type: String,
+  },
+	address: {
+    type: String,
+  },
+	description: {
+    type: String,
+  },
+	availability: [
```

Comment:
This design for availability may not fit the use cases of our project but don't worry as we will be revisiting this at a later date 😄

Reply:
My thinking was that I would do the necessary formatting in the frontend